src/ui/GenioWindow.cpp

```diff
@@ -1522,15 +1559,13 @@ GenioWindow::_FileSave(int32 index)
 	}
 */
 	// Stop monitoring if needed
-	if (editor->StopMonitoring() != B_OK)
```

Comment:
Here we are removing a previous change to log an error in case of stop monitor failure

Reply:
fixed